### PR TITLE
Fix CA1003: Use generic event handler instances

### PR DESCRIPTION
**Pull request by xtqqczze**
https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1003

This PR does not contain code fixes for public API surfaces, so the changes should not be breaking.

### CODE REVIEW

File: .globalconfig [hunk 1]
```diff
@@ -10,7 +10,8 @@ dotnet_diagnostic.CA1001.severity = silent
 dotnet_diagnostic.CA1002.severity = none
 
 # CA1003: Use generic event handler instances
-dotnet_diagnostic.CA1003.severity = none
+dotnet_diagnostic.CA1003.severity = warning
+dotnet_code_quality.ca1003.api_surface = private, internal
```
Comment (iSazonov) [inline .globalconfig:14]: Please move this in GlobalConfig for consistency.

Comment (xtqqczze) [reply .globalconfig:14]: move to where?

Comment (iSazonov) [reply .globalconfig:14]: https://github.com/PowerShell/PowerShell/blob/72608fb829881727f7d9b6295fb133613a30b6fe/.globalconfig#L12-L13

Comment (xtqqczze) [reply .globalconfig:14]: This is already in `.globalconfig`?

**Review by PaulHigin — APPROVED** [review]
LGTM

**Review by iSazonov — APPROVED** [review]

**Pull request merged**